pkg/audit/manager.go

```diff
@@ -235,6 +263,9 @@ func (am *Manager) audit(ctx context.Context) error {
 	for gvknn := range updateLists {
 		ar := updateLists[gvknn][0]
 		logConstraint(am.log, &gvknn, ar.enforcementAction, totalViolationsPerConstraint[gvknn])
+		if err := am.pubsubSystem.Publish(context.Background(), *auditConnection, *auditChannel, constraintMsg(&gvknn, ar.enforcementAction, totalViolationsPerConstraint[gvknn], timestamp)); err != nil {
```

Comment:
Do we want to publish different kinds of messages in the same channel? If I am consuming violations to raise alerts, I may want a violation-only channel.

What use cases are we targeting for audit events?

Reply:
Cool! For now, we can just stick to audit violations. And then we can come up with design to handle these kinds of situations of having multiple channels

Reply:
That means deleting this line, right?

Reply:
Yes.